earth2mip/inference_ensemble.py

```diff
@@ -258,7 +258,19 @@ def perturb(x, rank, batch_id, device):
             )
         if rank == 0 and batch_id == 0:  # first ens-member is deterministic
             noise[0, :, :, :, :] = 0
-        x += noise
+
+        if config.ic_perturbed_channels == "All":
+            x += noise
+        elif config.ic_perturbed_channels == "None":
+            pass
+        else:
+            if not isinstance(config.ic_perturbed_channels, list):
+                config.ic_perturbed_channels = [config.ic_perturbed_channels]
+            channel_list = model.channel_set.list_channels()
+            indices = torch.tensor(
+                [channel_list.index(channel) for channel in
+                 config.ic_perturbed_channels])
```

Comment:
What if someone provides a channel that is not in the channel list. Something list this should work?

```
 [channel_list.index(channel) if channel in channel_list for channel in config.ic_perturbed_channels]
```